### PR TITLE
Adds js to excluded scheme

### LINKED ISSUE #12: Javascript in links
Currently a link like `<a href="javascript:void(0);">` will resolve as `null` which means it indicates a broken link. In cases where a link begins with a scripting tag, the link should be treated as a success or ignored and not queried.

### PR DESCRIPTION
**Pull request by katerberg**
Fixes #12

### CODE REVIEW

File: lib/internal/defaultOptions.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ var userAgent = require("default-user-agent");
 var defaultOptions = 
 {
 	acceptedSchemes: ["http","https"],
-	excludedSchemes: ["data","geo","mailto","sms","tel"],
+	excludedSchemes: ["data","geo","mailto","sms","tel","javascript"],
```
Comment (stevenvachon) [inline lib/internal/defaultOptions.js]: alphabetical order, please

File: test/5.public.HtmlChecker.js [hunk 2]
```diff
@@ -233,13 +233,14 @@ describe("PUBLIC -- HtmlChecker", function()
 		
 		
 		
-		it('excludedSchemes = ["data","geo","mailto","sms","tel"]', function(done)
+		it('excludedSchemes = ["data","geo","mailto","sms","tel","javascript"]', function(done)
```
Comment (stevenvachon) [inline test/5.public.HtmlChecker.js]: ditto

File: test/5.public.HtmlChecker.js [hunk 3]
```diff
@@ -233,13 +233,14 @@ describe("PUBLIC -- HtmlChecker", function()
...
 		{
 			var htmlString = '<a href="data:image/gif;base64,R0lGODdhAQABAPAAAP///wAAACH/C1hNUCBEYXRhWE1QAz94cAAsAAAAAAEAAQAAAgJEAQA7">link1</a>';
 			htmlString += '<a href="geo:0,0">link2</a>';
 			htmlString += '<a href="mailto:address@email.com?subject=hello">link3</a>';
 			htmlString += '<a href="sms:+5-555-555-5555?body=hello">link4</a>';
-			htmlString += '<a href="tel:+5-555-555-5555">link5</a>';
+			htmlString += '<a href="javascript:void(0);">link5</a>';
```
Comment (stevenvachon) [inline test/5.public.HtmlChecker.js]: ditto

**Comment by katerberg** [comment]
Fixed alphabetization.

File: test/5.public.HtmlChecker.js [hunk 4]
```diff
@@ -233,13 +233,14 @@ describe("PUBLIC -- HtmlChecker", function()
 		
 		
 		
-		it('excludedSchemes = ["data","geo","mailto","sms","tel"]', function(done)
+		it('excludedSchemes = ["data","geo","javascript","mailto","sms","tel"]', function(done)
 		{
 			var htmlString = '<a href="data:image/gif;base64,R0lGODdhAQABAPAAAP///wAAACH/C1hNUCBEYXRhWE1QAz94cAAsAAAAAAEAAQAAAgJEAQA7">link1</a>';
 			htmlString += '<a href="geo:0,0">link2</a>';
+			htmlString += '<a href="javascript:void(0);">link5</a>';
```
Comment (stevenvachon) [inline test/5.public.HtmlChecker.js]: "link3", and following link text adjustments as well

**Comment by katerberg** [comment]
Done

**Comment by stevenvachon** [comment]
Oh, readme needs to be updated as well

**Comment by katerberg** [comment]
I swear one of these PRs, I'm going to figure out this project. Sorry for all the ping pongs.

**Comment by stevenvachon** [comment]
Looks great! Thanks!!

**Comment by stevenvachon** [comment]
:) np

basically anything that can help avoid future thinking/comprehension

**Pull request merged**